package-versions/grouping-and-overrides/multiple-groups.md

```diff
@@ -0,0 +1,120 @@
+        <!-- baz has a dependency on foo 303.0.0 -->
+        <!-- baz has a dependency on qux 303.0.0 -->
+        <!-- baz has a dependency on xyzzy 303.0.0 -->
+        <PackageVersion Include="baz" Version="30.0.0" CentralManagementGroup="B" />
+    </ItemGroup>
+</Project>
+```
+
+In this `Directory.Packages.props` file, there are three `CentralManagementGroup` of `<PackageVersion>` defined.
+
+1. The global `CentralManagementGroup` is automatically created, and all `<PackageVersion>` that do not have the `CentralManagementGroup` attribute defined belong to this group.
+2. `CentralManagementGroup="A"` defines a `CentralManagementGroup` with `<PackageVersion>` for 5 packages.
+3. `CentralManagementGroup="B"` defines a `CentralManagementGroup` with `<PackageVersion>` for 3 packages.
+
+*Example Package to Demonstrate Version Override*
+
+There are two `<PackageVersion>` defined for `Newtonsoft.Json` to demonstrate using both the global and a named `CentralManagementGroup`.
+
+- If a `<PackageReference>` does not specify a `CentralManagementGroup`, it will import version 12.0.2 from the global group.
+- If `CentralManagementGroup=A` is defined, it will import 11.0.2 from group A.
+
+### Sample Project 6: Multiple Named Groups
+
+#### 6.csproj
+
+```xml
+<Project>
+  <ItemGroup>
+    <!-- global group -->
+    <!-- Version 2.9.0 -->
+    <PackageReference Include="Serilog"/>
+    <!-- "A" group -->
+    <!-- Version 11.0.2 -->
+    <PackageReference Include="Newtonsoft.Json" CentralManagementGroup="A" />
+    <!-- "B" group -->
+    <!-- Version 3.3.3 -->
+    <PackageReference Include="foo" CentralManagementGroup="B" />
+  </ItemGroup>
+</Project>
+```
+
+This project uses three  `CentralManagementGroup`, `"A",` `"B"`, and the global group.
+
+All `<PackageReference>` dependencies are included between these `CentralManagementGroup` and are directly imported.
+
+### Sample Project 7: Multiple Named Groups with Transitive Dependencies
+
+#### 7.csproj
+
+```xml
+<Project>
+  <ItemGroup>
+    <!-- global group -->
+    <!-- Version 2.9.0 -->
+    <PackageReference Include="Serilog"/>
+    <!-- "A" group -->
+    <!-- Version 11.0.2 -->
+    <PackageReference Include="Newtonsoft.Json" CentralManagementGroup="A" />
+    <!-- "A" group -->
+    <!-- Version 20.0.0 -->
+    <PackageReference Include="bar" CentralManagementGroup="A" />
+    <!-- "B" group -->
+    <!-- Version 3.3.3 -->
+    <PackageReference Include="foo" CentralManagementGroup="B" />
+  </ItemGroup>
+</Project>
+```
+
+This project uses three  `CentralManagementGroup`, `"A",` `"B"`, and the global group.
+
+`bar` has three dependencies. These are transitive dependencies for the project.
+
+- `foo` is imported with version 3.3.3 from **`CentralManagementGroup="B"` because `foo` is directly imported**.
+- `qux` is imported with version 2.0.0 from `CentralManagementGroup="A"` because `bar` is imported from the same group.
```

Comment:
1. What would happen if the qux is not in Group A but global? Will the version from global be used? 
2. What would happen if bar is not in Group A but global? Conform with the [fallback rule](https://github.com/rconard/central-nuget-management-planning/blob/9f88f93a8a213e9c764a939346d2290292e71109/package-versions/grouping-and-overrides/transitives.md#single-group-with-global-fallback) its version will be from the global group. Will the qux Group A version be used? 
3. We could imagine more transitive dependencies jumping between Global and Group A will picking the version from Global or Group A be ok?